### PR TITLE
IBX-5022 - Checkout TWIG functions

### PR DESCRIPTION
**Pull request by dabrt**
| Question      | Answer |
| ------------- | --- |
| JIRA Ticket   | IBX-5022 |
| Versions      | 4.4 and up |
| Edition       | Commerce |

Add a list of Twig functions exposed by the Checkout component

#### Checklist

- [x] Text renders correctly
- [x] Text has been checked with vale
- [x] Description metadata is up to date
- [x] Added link to this PR in relevant JIRA ticket or code PR

### CODE REVIEW

File: docs/templating/twig_function_reference/checkout_twig_functions.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
+---
+description: Checkout Twig return information about the checkout process, and total values related to cart and cart items.
```
Comment (konradoboza) [inline docs/templating/twig_function_reference/checkout_twig_functions.md:2]: Shouldn't it be:
```suggestion
description: Checkout Twig functions return information about the checkout process, and total values related to cart and cart items.
```
?

File: docs/templating/twig_function_reference/checkout_twig_functions.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+``` html+twig 
+<a href="{{ ibexa_checkout_step_path(checkout, step) }}">{{ <link_label> }}</a>
+```
+
+### `ibexa_checkout_step_url()`
+
+The `ibexa_checkout_step_url()` function returns a URL address of the step.
+
+``` html+twig 
+<a href="{{ ibexa_checkout_step_url(checkout, step) }}">{{ <target_page_label> }}</a>
```
Comment (konradoboza) [inline docs/templating/twig_function_reference/checkout_twig_functions.md:44]: There is also a third parameter '$schemaRelative = false` by default which if specified (true/false), generates relative or absolute URL.

File: docs/templating/twig_function_reference/checkout_twig_functions.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+``` html+twig 
+<a href="{{ ibexa_checkout_step_path(checkout, step) }}">{{ <link_label> }}</a>
+```
+
+### `ibexa_checkout_step_url()`
+
+The `ibexa_checkout_step_url()` function returns a URL address of the step. 
+By setting the optional argument to `true` you can decide whether the function 
+returns a relative or absolute URL of the checkout step.
+The default value of the optional argument is `false`.
```
Comment (konradoboza) [inline docs/templating/twig_function_reference/checkout_twig_functions.md:44]:
```suggestion
The default value of the optional argument is `false` which stands for absolute URL.
```

**Review by konradoboza — APPROVED** [review]

File: docs/templating/twig_function_reference/checkout_twig_functions.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
+---
+description: Checkout Twig functions return information about the checkout process, and total values related to cart and cart items.
+edition: commerce
+---
+
+# Checkout Twig functions
+
+You can use Checkout Twig functions to get information about the checkout process,
```
Comment (juskora) [inline docs/templating/twig_function_reference/checkout_twig_functions.md:8]:
```suggestion
You can use checkout Twig functions to get information about the checkout process, 
```

File: docs/templating/twig_function_reference/checkout_twig_functions.md [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+---
+
+# Checkout Twig functions
+
+You can use Checkout Twig functions to get information about the checkout process, 
+as well as total values related to cart and cart items.
+
+### `ibexa_checkout_step_label()`
+
+The 'ibexa_checkout_step_label()' function returns a name of the step (configured in `framework.workflows.workflow.ibexa_checkout.transitions.<transition>.metadata.label`).
```
Comment (juskora) [inline docs/templating/twig_function_reference/checkout_twig_functions.md:13]:
```suggestion
The `ibexa_checkout_step_label()` function returns a name of the step (configured in `framework.workflows.workflow.ibexa_checkout.transitions.<transition>.metadata.label`).
```

File: docs/templating/twig_function_reference/checkout_twig_functions.md [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+{% block items %}
+    {% for entry in ibexa_checkout_summary_entries(summary) %}
+        // ...
+    {% endfor %}
+{% endblock %}  
+```
+
+### `ibexa_checkout_summary_vat_summaries()`
+
+'The `ibexa_checkout_summary_vat_summaries()` function takes in a single argument, a cart summary object,
```
Comment (juskora) [inline docs/templating/twig_function_reference/checkout_twig_functions.md:74]:
```suggestion
The `ibexa_checkout_summary_vat_summaries()` function takes in a single argument, a cart summary object, 
```

File: docs/templating/twig_function_reference/twig_function_reference.md [hunk 7]
```diff
@@ -10,6 +10,7 @@ and [Twig extensions provided by Symfony]([[= symfony_doc =]]/reference/twig_ref
 
 [[= cards([
   "templating/twig_function_reference/cart_twig_functions",
+  "templating/twig_function_reference/checkout_twig_functions",
```
Comment (juskora) [inline docs/templating/twig_function_reference/twig_function_reference.md:13]: Broken link.

**Review by juskora — CHANGES_REQUESTED** [review]

**Review by juskora — APPROVED** [review]

**Pull request merged**